apis/core.oam.dev/v1beta1/core_types.go

```diff
@@ -242,6 +243,34 @@ type ScopeDefinitionList struct {
 type ResourceTracker struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Status ResourceTrackerStatus `json:"status,omitempty"`
+}
+
+// ResourceTrackerStatus define the status of resourceTracker
+type ResourceTrackerStatus struct {
+	TrackedResource []TypedReference `json:"trackedresources,omitempty"`
+}
+
+// A TypedReference refers to an object by Name, Kind, and APIVersion. It is
+// commonly used to reference cross namespace objects
```

Comment:
```suggestion
// commonly used to reference objects outside the application namespace.
```

The word `cross namespace` is not right/accurate to represent your feature, please check and find a new one.

Reply:
`across namespace` is better.